mods/ra/sequences/aircraft.yaml

```diff
@@ -70,3 +70,17 @@ u2:
 badr:
 	idle:
 		Facings: 16
+		
```

Comment:
Please revert this change.

Reply:
There's nothing to be reverted as the mh60 sequnce is right behind it. I will delete the unnecessary tabs